server/src/test/java/io/spine/server/aggregate/AggregateRepositoryShould.java

```diff
@@ -192,6 +233,12 @@ public void allow_to_change_snapshot_trigger() {
     }
 
     @Test
+    public void not_update_snapshot_trigger_for_unassigned_storage() {
+        final ProjectAggregateRepository repoWithoutStorage = new ProjectAggregateRepository();
+        repoWithoutStorage.setSnapshotTrigger(10);
```

Comment:
I don't understand this test. There is no expected exceptions. And I don't see any assertions.